ci-operator/config/cloud-bulldozer/e2e-benchmarking/cloud-bulldozer-e2e-benchmarking-master.yaml

```diff
@@ -288,6 +288,29 @@ tests:
     test:
     - ref: cloud-bulldozer-e2e-benchmarking-udn-density-pods
     workflow: openshift-qe-installer-aws
+- always_run: false
+  as: e2e-rds-core-single-node
+  capabilities:
+  - intranet
+  restrict_network_access: false
+  steps:
+    cluster_profile: metal-perscale-cpt
```

Comment:
I dont think this is the right place to test this. A better candidate would be the jetlag ci cluster:
```
cluster_profile: metal-perfscale-jetlag
```

Also you to move to the jetlag CI cluster add need to add some envs:
```
      BASTION_CP_INTERFACE: ens7f1
      LAB_CLOUD: cloud19
      LAB_INTERFACE: eno3
      NUM_WORKER_NODES: "2"
```

cc @jtaleric @akrzos @radez

Reply:
Also, may need to overwrite some vars to adjust the SRIOV and the perf-profile setup to the jetlag CI cluster

Reply:
@josecastillolema may I know why are you suggestion jetlag CI over this one? If this is aiming movement of all our current jobs, can we take it in a different PR?

Reply:
Understood and acknowledged.